docker-compose.yml

```diff
@@ -2,82 +2,50 @@ version: '3.1'
 
 services:
   bwm:
-    image: racoon/bandwidth-monitor
+    image: racoon/bandwidth-monitor:nightly
     restart: always
     depends_on: 
-      - influx-db
+      - influx
     networks:
       - bwm
     environment:
-      DBTYPE: influx
-      DBHOST: influx-db
+      DBTYPE: influxdb
+      DBHOST: influx
       DBUSER: bwm
       DBPASSWORD: bwm
-  influx-db:
+    volumes:
+      - ./log:/bwm/log
+      - ./lib:/bwm/lib
+  influx:
     image: influxdb:latest
-    ports:
-      - '8086:8086'
+    networks:
+      - bwm
     volumes:
       - influxdb-storage:/var/lib/influxdb
     environment:
-      - INFLUXDB_DB=bwm-db
-      - INFLUXDB_USER=${INFLUXDB_USERNAME}
-      - INFLUXDB_PASSWORD=${INFLUXDB_PASSWORD}
-  chronograf:
-    image: chronograf:latest
-    ports:
-      - '8888:8888'
-    volumes:
-      - chronograf-storage:/var/lib/chronograf
-    depends_on:
-      - influxdb
-    environment:
-      - INFLUXDB_URL=http://influxdb:8086
-      - INFLUXDB_USERNAME=${INFLUXDB_USERNAME}
-      - INFLUXDB_PASSWORD=${INFLUXDB_PASSWORD}
+      - INFLUXDB_DB=bwm
+      - INFLUXDB_USER=bwm
+      - INFLUXDB_PASSWORD=bwm
   grafana:
     image: grafana/grafana:latest
     ports:
       - '3000:3000'
+    networks:
+      - bwm
     volumes:
       - grafana-storage:/var/lib/grafana
       - ./grafana-provisioning/:/etc/grafana/provisioning
```

Comment:
Can this be included in one of the docker volumes? If yes, please do so.

Reply:
grafana-provisioning held config file for data source and dashboards. On start time grafana takes it and load it. 
Advantage you can change it localy and then restart the docker-compose and the dashboard will be changed automaticaly